website/docs/language/settings/terraform-cloud.html.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "language"
+page_title: "Terraform Cloud Configuration - Terraform Settings - Configuration Language"
+sidebar_current: "docs-config-terraform"
+description: "The nested `cloud` block configures Terraform's integration with Terraform Cloud."
```

Comment:
should we include the word "state" somewhere in here to make sure that people searching for backends or state storage find TFC